core/tests/test_utils.py

```diff
@@ -1076,6 +1076,12 @@ def save_new_valid_exploration(
             language_code=language_code)
         exploration.states[exploration.init_state_name].update_interaction_id(
             interaction_id)
+
+        if interaction_id == 'EndExploration':
```

Comment:
Ah no, I meant to let the first interaction be something and then use the `end_state_name` field to set the end exploration so the created exploration has 2 states.
Basically, no need to add this block as the same functionality already exists.

Reply:
I understand. I've made the changes.